packages/crayons-core/src/components/data-table/readme.md

```diff
@@ -658,6 +658,158 @@ function App() {
 </code-block>
 </code-group>
 
+
+#### Collection variant
+
+Row value for this column variant should be an object with the following properties:
+
+1. collection - Array of strings.
+2. show (optional) - Number of items to show from collection initially. Default value is 2.
+
+```html live
+  <fw-data-table id="datatable-33" label="Data table 33">
+  </fw-data-table>
+
+  <script type="application/javascript">
+    var data = {
+      columns: [{
+        "key": "objectname",
+        "text": "Object Name"
+      }, {
+        "key": "related",
+        "text": "Related Objects",
+        "variant": "collection",
+        "widthProperties": {
+          "width": "400px"
+        },
+      },
+      {
+        "key": "count",
+        "text": "Count"
+      }],
+      rows: [{
+        "id": "0011",
+        "objectname": "Library",
+        "related": { 
+          "collection": ["Books", "Authors", "Locations", "Tags"],
+          "show": 2
```

Comment:
I feel it's better if we keep `show` as a configuration at the column level instead of per row level. Ideally, the consumer of this, might not be knowing the length of collection for each row, since it is dynamic.

Reply:
I agree. Added a variantProperties in column configuration. This can be used to pass column level properties to the variants.